README.rst

```diff
@@ -503,8 +503,83 @@ Models
+                  #. ``Specify the maximum allowed percentage of frames above the framewise
+                     displacement threshold of (FD max from above)``
+
+               -  ``No`` All subjects are retained for analyses regardless of movement.
+
+         -  ``Linear model`` This will apply a linear model that captures main and/or interaction effects of one or more variables that must be present in a covariate/group data spreadsheet file. This file must contain a column with subject ID names identical to those used for the functional data.
+
+            #. ``Specify model name``
+            #. ``Aggregate scan-level statistics before analysis?`` See above
+            #. ``Specify the path of the covariates/group data spreadsheet file`` Can be a .csv, .txt, .tsv, or .xlsx file
+
+               #. ``Specify the column containing subject names``
+               #. ``Specify the column data types`` Note that the black option is selected, the grey option is not. Toggle left/right keys to select or deselect options and up/down keys to move through the list of variables in your spreadsheet file.
+               #. ``Specify the subjects to use`` Select the subjects to include in this analysis by their categorical variables. For multiple categorical variables, the intersection of the groups will be used.
+
+            #. ``Exclude subjects based on movement?`` See above
+
+               -  ``Yes``
+
+                  #. ``Specify the maximum allowed mean framewise displacement in mm``
+                  #. ``Specify the maximum allowed percentage of frames above the framewise
+                     displacement threshold of (FD max from above)``
+
+               -  ``No`` Continue
+
+            #. ``Specify the variables to add to the model`` For all variables that are selected here, the model will output a main effect, controlling for all other variables selected by treating them as covariates of no interest. Anything not selected at this stage will be left entirely out of the model.
+
+               -  ``Listwise deletion`` Will remove subjects with a missing value from analysis entirely.
+               -  ``Mean substitution`` For continuous variables, will impute the average of all subjects with a valid observation for this variable.
+
+            #. ``Specify additional contrasts for categorical variables?`` Contrasts for the mean across all subjects, and for all variables will be generated automatically. If you would like to output additional contrasts between particular groups, this should be specified here.
```

Comment:
I would elaborate this a bit. 
For each variable, we generate one contrast. This means that we, by default, only have the omnibus F-test for categorical variables with more than two levels.

Reply:
Resolved